src/legacy/ui/ui_settings/ui_settings_service_for_request.ts

```diff
@@ -36,16 +35,8 @@ type Options = Pick<UiSettingsServiceOptions, 'getDefaults' | 'overrides'>;
  */
 export function getUiSettingsServiceForRequest(
   server: Legacy.Server,
-  request: Legacy.Request,
-  options: Options
+  request: Legacy.Request
 ): IUiSettingsClient {
-  const { getDefaults, overrides } = options;
-
-  const uiSettingsService = uiSettingsServiceFactory(server, {
-    getDefaults,
-    overrides,
-    savedObjectsClient: request.getSavedObjectsClient(),
-  });
-
-  return uiSettingsService;
+  const savedObjectsClient = request.getSavedObjectsClient();
+  return uiSettingsServiceFactory(server, { savedObjectsClient });
```

Comment:
I left uiSettings routes and integration tests in LP.https://github.com/restrry/kibana/tree/7493525b4563cdbece7a46aff8cb17f7ed5e9611/src/legacy/ui/ui_settings/routes

 will move them in a separate PR.